java/src/apps/gui3/FirstTimeStartUpWizard.java

```diff
@@ -309,6 +316,11 @@ public void run() {
             InstanceManager.getDefault(RosterConfigManager.class).savePreferences(project);
             InstanceManager.getDefault(GuiLafPreferencesManager.class).savePreferences(project);
             connectionConfigPane.savePreferences();
+            try {
+                InstanceManager.getDefault(WarrantPreferences.class).initialize(project);
+            } catch ( InitializationException ex ){
+                log.error("Exception Initialising warrant preserences: ", ex);
```

Comment:
"preserences" is misspelled.

Reply:
Why does warrant preferences need to be initialized here?

Reply:
Thanks for the typo catch.
See #11617 for FTSW Warrant Preference issues.